parser.y

```diff
@@ -2767,6 +2777,26 @@ ColumnNameListOpt:
 	}
 |	ColumnNameList
 
+IdentListWithParenOpt:
```

Comment:
```suggestion
IdentListWithParentOpt: ？
```

Reply:
Not parent. paren means '('.
See:
LikeTableWithOrWithoutParen:
	"LIKE" TableName
	{
		$$ = $2
	}
|	'(' "LIKE" TableName ')'
	{
		$$ = $3
	}